wazuh/wazuh-kibana-app
# Delete wazuh token called twice

Hi team, this resolves 
- Delete wazuh token is called twice

Testing:
- Rollback the changes, add a console.log in [public/react-services/wz-authentication.ts#L151](https://github.com/wazuh/wazuh-kibana-app/blob/e22abcb81e2543d92f00c392785b11e5a981bbc0/public/react-services/wz-authentication.ts#L151)
- Run the logout and see the browser console

**To test it:**

**Precondition 1**: Location: [public/react-services/wz-authentication.ts#L151](https://github.com/wazuh/wazuh-kibana-app/blob/e22abcb81e2543d92f00c392785b11e5a981bbc0/public/react-services/wz-authentication.ts#L151)

1. Go to Inventory Section of an agent
- **Given** the browser is logged in the Wazuh kibana app
- **When** the put a console log in **Precondition 1**
- **And** the user will logged out
- **Then** the console log will appear properly in the browser console
